README.md

```diff
@@ -1,3 +1,4 @@
 # testx
 
 1
+2
```

Comment:
3 comes after 2

Reply:
could you please, click on Request  Changes?
![image](https://user-images.githubusercontent.com/76909783/115580771-12f6e600-a2d0-11eb-8824-51c4b641427c.png)